README.md

```diff
@@ -101,11 +114,11 @@ State Channels offer an innovative solution, where off-chain transactions are ex
 
 State channels protocol inherently offers security to the demo.
 
-State Channel Transactions are ruled by co-signed smart contracts. Each pariticipant shall verify the content of a transaction before signing it. Those transactions can be off-chain contract calls, spend transactions or on-chain such as `MutualCloseChannelTx`. All transactions need to be co-signed by both parties.
+State Channel Transactions are ruled by co-signed smart contracts. Each participant shall verify the content of a transaction before signing it. Those transactions can be off-chain contract calls, spend transactions or on-chain such as `MutualCloseChannelTx`. All transactions need to be co-signed by both parties.
```

Comment:
Strictly speaking, in state channels you have `update` operations, which can be `transfer` (similar to `spend` on-chain), `deposit`, `withdraw`, `new_contract`, `call_contract` (and `meta`, which basically adds a comment to another op).